spec/services/agency_seeder_spec.rb

```diff
@@ -6,10 +6,19 @@
   let(:deploy_env) { 'int' }
 
   describe '#run' do
-    before { Agency.delete_all }
+    before do
+      Agency.delete_all
+      mock_root = instance_spy(Pathname)
+      fixture = Rails.root.join('spec', 'fixtures', 'agencies.yml')
+      allow(Rails).to receive(:root).and_return(mock_root)
```

Comment:
The stubbing here makes me wonder if it's time for a refactor of the AgencySeeder so we can use dependency injection instead of stubbing. We could init it with a file path and if none is given have it assume the current one as the default.

Reply:
That would definitely be cleaner - I'll take a stab at it and include it if it's quick.